website/settings/_base.py

```diff
@@ -58,8 +58,8 @@
         'NAME': 'postgres',
         'USER': 'postgres',
         'PASSWORD': '',
-        'HOST': 'localhost',
-        'PORT': 5432,
+        'HOST': os.getenv('POSTGRES_HOST', 'localhost'),
```

Comment:
Since we aren't using any postgres specific stuff, would it be best to use `DATABASE_{HOST,PORT}` since the ORM supports others? This is fine, but would let us be consistent across projects if sirbot does the same.